src/stock-operations/stock-operations-table.component.tsx

```diff
@@ -298,9 +336,36 @@ const StockOperations: React.FC<StockOperationsTableProps> = () => {
               }}
             >
               <TableToolbarContent className={styles.toolbarContent}>
+                <Layer style={{ margin: "4px", display: "flex" }}>
```

Comment:
can we have this in a separate component

Reply:
> can we have this in a separate component

This has been resolved in the subsequent commits following this